mono/mini/exceptions-arm.c

```diff
@@ -385,28 +385,42 @@ mono_arch_get_throw_corlib_exception (MonoTrampInfo **info, gboolean aot)
 }	
 
 GSList*
-mono_arm_get_exception_trampolines (gboolean aot)
+mono_arm_get_exception_trampoline (gboolean corlib, gboolean rethrow, gboolean llvm, gboolean resume_unwind,
+				   const char *tramp_name, gboolean aot, gboolean reg, GSList *tramps)
 {
 	MonoTrampInfo *info;
-	GSList *tramps = NULL;
 
-	/* LLVM uses the normal trampolines, but with a different name */
-	get_throw_trampoline (168, TRUE, FALSE, FALSE, FALSE, "llvm_throw_corlib_exception_trampoline", &info, aot, FALSE);
-	tramps = g_slist_prepend (tramps, info);
-	
-	get_throw_trampoline (168, TRUE, FALSE, TRUE, FALSE, "llvm_throw_corlib_exception_abs_trampoline", &info, aot, FALSE);
-	tramps = g_slist_prepend (tramps, info);
+	(void)get_throw_trampoline (168, corlib, rethrow, llvm, resume_unwind, tramp_name, &info, aot, FALSE);
```

Comment:
this should be a named constant instead of a magic number

Reply:
I don't disagree, but that is what it already was.